webapp/locales/de.json

```diff
@@ -377,9 +378,11 @@
   },
   "filter-menu": {
     "all": "Alle",
+    "article": "Artikel",
     "categories": "Themen",
     "deleteFilter": "Filter löschen",
     "emotions": "Emotionen",
+    "events": "Veranstaltungen",
```

Comment:
where is the plural for `article`?

Reply:
![image](https://github.com/Ocelot-Social-Community/Ocelot-Social/assets/1238238/6ec0f6ee-cd84-43d0-b2d6-fee0aa97ac7f)

Reply:
![image](https://github.com/Ocelot-Social-Community/Ocelot-Social/assets/1238238/5d197a7a-8822-497b-af0e-9f71f2d80c13)